packages/rum-core/src/performance-monitoring/span-base.js

```diff
@@ -48,16 +48,14 @@ class SpanBase {
     }
   }
 
-  addTags(tags) {
+  addLabels(labels) {
```

Comment:
Maybe we should deprecated `addTags` in this minor version and remove it in the next major. WDYT?